docs/docsite/rst/os_guide/windows_winrm.rst

```diff
@@ -21,7 +21,16 @@ configuration is required to use WinRM with Ansible.
 
 Ansible uses the `pywinrm <https://github.com/diyan/pywinrm>`_ package to
 communicate with Windows servers over WinRM. It is not installed by default
-with the Ansible package, but can be installed by running the following:
+with the Ansible package.
+
+If you chose the `pipx` install instructions, you can install it by running the
+following:
+
+.. code-block:: shell
+
+   pipx inject ansible pywinrm
```

Comment:
Hmm, this is somewhat problematic if users installed ansible-core instead of ansible with pipx (see https://docs.ansible.com/ansible/devel/installation_guide/intro_installation.html). I'm not sure I have a good idea for what to do in that situation though (except having even more branching)...

Reply:
That is a good point. Are you thinking in terms of the fact that they would need to do `pipx inject ansible-core pywinrm` or is there more to it than that? (I know they would need to install the Windows collection manually etc, but I think that would be its own situation).

I think on the main intro guide, we don't account for the `ansible-core` situation:
- https://docs.ansible.com/ansible/devel/installation_guide/intro_installation.html#installing-extra-python-dependencies
- https://docs.ansible.com/ansible/devel/installation_guide/intro_installation.html#installing-argcomplete

Reply:
> Are you thinking in terms of the fact that they would need to do `pipx inject ansible-core pywinrm` or is there more to it than that?

Yes, I was thinking about that, and yes, I'm aware that other parts of the docs also ignore this :) I'm not really sure what to do here. The best solution is probably to provide both command lines. How about having a comment to distinguish them? Something like:
```suggestion
   pipx inject ansible pywinrm  # if you installed ansible with pipx
   pipx inject ansible-core pywinrm  # if you installed ansible-core with pipx
```

Reply:
That makes a lot of sense as a solution!